frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDetailsActions.tsx

```diff
@@ -78,6 +93,32 @@ const PipelineRunDetailsActions: React.FC<PipelineRunDetailsActionsProps> = ({ o
               ) : (
                 <React.Fragment key="compare-runs" />
               ),
+              !isExperimentActive ? (
```

Comment:
the logic is still wrong here. This check here should be if the run is not active. The the check to see if the item is disabled would be what you have now, isExperimentActive. And you still need to not show the tooltip when it is not disabled